rust_src/src/fonts.rs

```diff
@@ -131,4 +133,15 @@ pub fn font_match_p(spec: LispObject, font: LispObject) -> bool {
     unsafe { c_font_match_p(spec, font) }
 }
 
+/// Return a font-entity matching with FONT-SPEC on the current frame.
+/// Optional 2nd argument FRAME, if non-nil, specifies the target frame.
+#[lisp_fn]
```

Comment:
There is a tool in admin/ called `rustize-emacs`. Give it a go. If the second argument can be left out then this needs a `min`.
```suggestion
#[lisp_fn(min = "1")]
```

Reply:
Yep. Forgot that.